app/views/shared/_h1_heading.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= content_for(:page_title) %></h1>
+  </div>
+</div>
```

Comment:
From what Simon told me, the whole page should be in  2/3rd div except the error summary.
And it should be in this order : 

```
Error summary (full width)

<div 2 third >

  Title

  Form

</div>
```

Like on the **Enter your client's details** page

Reply:
I think @Kalagan is right here.

I can understand why you may want to dry up the repeated heading entries, but considering Julien's comments, I would suggest that you do it via a helper method, and that it just does the `h1` tag. Something like:
```ruby
def h1_for_content(content_name, html_class: 'govuk-heading-xl')
  content_tag :h1, content_for(content_name), class: html_class
end
```
Then in your code you'd enter: `<%= h1_for_content(:page_title) %>` and I think it will be succinct, but still clear what's being entered